### ISSUE TITLE
Doesn't work anymore

### ISSUE THREAD

**Issue by Miloune** [GhostFlying/NotificationManagerXposed#12]
Hello, 
I'm on Oneplus 3 with Android 6.0.1 and using xposed v86. 
Your application look great and I want to use it, but doesn't work for me. 
Is it possible to update or fix it? Please 
Thanks a lot

**Comment by GhostFlying** [comment]
I will check it some days later but as nowadays I am preparing some
interviews, it may take more time.

**Comment by Miloune** [comment]
Thanks a lot, have a good luck ☺️

**Comment by GhostFlying** [comment]
Sorry for the delay. What's the app you tried to hook and have you tried restart the app?

**Comment by Miloune** [comment]
I'm sorry, didn't remember that, but i think it's on all apps. I restarded all, Phone + Apps.

**Comment by damionx7** [comment]
Yeah it doesnt work . Used on one plus 3 android 6.0.1 and app is boldbeast recorder .
